test/e2e/framework/framework.go

```diff
@@ -224,59 +222,42 @@ func IsResourceAvailable(errChan chan error, client *ClientSet, resource Testing
 	errChan <- err
 }
 
-// checks 3 times if the resources are unavailable
-// - is fine if fails or 1st or 2nd run, resources could be in the process of being removed
-// - is not fine if resources disappear, then reappear
-// - it seems to take a bit longer to remove resources, so this wrapper should account for that.
-func ConsoleResourcesUnavailable(client *ClientSet) error {
-	var failed error = nil
-	// give it 3 tries, then fail
-	for i := 0; i < 3; i++ {
-		// testing resources are hard-coded in this func.
-		err := LoopResources(client, IsResourceUnavailable)
-		fmt.Printf("validating console resources have been removed... %v\n", err == nil)
-		failed = err
-	}
-	return failed
-}
-
-func LoopResources(client *ClientSet, inner func(errChan chan error, client *ClientSet, resource TestingResource)) error {
+func ConsoleResourcesUnavailable(t *testing.T, client *ClientSet) error {
 	resources := getTestingResources()
 
 	errChan := make(chan error)
 	for _, resource := range resources {
-		go inner(errChan, client, resource)
```

Comment:
where did `inner` function go?
is it still being used somewhere?

Reply:
Inner function was previously passed in as an argument, but since had only a single caller passing `IsResourceUnavailable` I decided to call it directly.